3d_armor_stand/init.lua

```diff
@@ -421,6 +322,26 @@ minetest.register_abm({
 	end
 })
 
+minetest.register_lbm({
+	label = "Update armor stand inventories",
+	name = "3d_armor_stand:update_inventories",
+	nodenames = {"3d_armor_stand:locked_armor_stand", "3d_armor_stand:shared_armor_stand", "3d_armor_stand:armor_stand"},
+	run_at_every_load = false,
+	action = function(pos, node)
+		local meta = minetest.get_meta(pos)
+		local inv = meta:get_inventory()
+		local lists = inv:get_lists()
+		inv:set_lists({main = {
+			lists.armor_head and lists.armor_head[1] or ItemStack(),
```

Comment:
Return early in case `lists.armor_*` does not exist, such that there's no item loss in case `env_meta.txt` is corrupted/deleted (it contains the LBM run information). Alternatively, you could use a node metadata field to do versioning.
This special handling is often not required in other mods because they replace the nodes within the LBM, thus never triggering it again.